docs/tutorials/settings_tutorial.ipynb

```diff
@@ -0,0 +1,437 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "settings.llm = llm_anthropic\n",
+    "settings.llm_config = {\n",
+    "    \"model_list\": [\n",
+    "        {\n",
+    "            \"model_name\": llm_anthropic,\n",
+    "            \"litellm_params\": {\n",
+    "                \"model\": llm_anthropic,\n",
+    "                \"temperature\": 0.1,\n",
+    "                \"max_tokens\": 512,\n",
+    "            }\n",
+    "        }\n",
+    "    ],\n",
+    "    \"rate_limit\": {\n",
+    "        llm_anthropic: \"30000 per 1 minute\",\n",
+    "    },\n",
+    "}\n",
+    "settings.summary_llm = llm_anthropic\n",
+    "settings.summary_llm_config = {\n",
+    "    \"rate_limit\": {\n",
+    "        llm_anthropic: \"30000 per 1 minute\",\n",
+    "    },\n",
+    "}\n",
+    "settings.agent = AgentSettings(\n",
+    "    agent_llm=llm_anthropic,\n",
+    "    agent_llm_config={\n",
+    "        \"rate_limit\": {\n",
+    "            llm_anthropic: \"30000 per 1 minute\",\n",
+    "        },\n",
+    "    },\n",
+    "    index=IndexSettings(\n",
+    "            paper_directory=pathlib.Path.cwd().joinpath(\"papers\"),\n",
+    "            index_directory=pathlib.Path.cwd().joinpath(\"papers/index\"),\n",
+    "        )\n",
+    ")\n",
+    "# settings.embedding = \"st-multi-qa-MiniLM-L6-cos-v1\"\n",
+    "settings.embedding = \"hybrid-st-multi-qa-MiniLM-L6-cos-v1\"\n",
+    "response = ask(\"What are the most relevant language models used for chemistry?\", settings=settings)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now the agent is able to use `Anthropic` models only and although we don't have a valid `OPENAI_API_KEY`, the question is answered because the agent will not use `OpenAI` models. See that we also changed the `embedding` because it was using `text-embedding-3-small` by default, which is a `OpenAI` model. `Paperqa` implements a few embedding models. Please refer to the [documentation](https://github.com/Future-House/paper-qa?tab=readme-ov-file#embedding-model) for more information.\n",
+    "\n",
+    "Notice that we redefined `settings.agent.paper_directory` and `settings.agent.index` settings. `Paperqa` actually uses the setting from `settings.agent`. However, for convenience, we implemented an alias in `settings.paper_directory` and `settings.index_directory`."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# The output\n",
+    "\n",
+    "`Paperqa` returns a `PQASession` object, which contains not only the answer but also all the information gatheres to answer the questions."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "response.__dict__"
```

Comment:
It feels unexpected to use `__dict__` (we don't even use this in our own code), do we need this cell, or can we use `print` with a message?